packages/rocm-k8s-device-plugin/rocm-k8s-device-plugin.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Start ROCm kubernetes device plugin
+After=kubelet.service
+Wants=kubelet.service
+
+[Service]
+# Ensure that the kubelet device plugin socket exists before we start
+# A brief sleep is needed to avoid the `test` failing its first check
+ExecStartPre=/usr/bin/sleep 0.1
+ExecStartPre=/usr/bin/test -S /var/lib/kubelet/device-plugins/kubelet.sock
+ExecStart=/usr/bin/rocm-device-plugin -logtostderr=true -stderrthreshold=INFO -v=5
```

Comment:
For other reviewers, all these flags are parsed automatically by Go's flag parsing library:

https://github.com/ROCm/k8s-device-plugin/blob/763445e18f3838fa72b22e31a04ec25987334bff/vendor/github.com/golang/glog/glog_flags.go#L397